package.json

```diff
@@ -32,30 +32,25 @@
         "csb:stop": "./core/scripts/stop.sh",
         "csb:storage": "./core/scripts/launch_storage.sh",
         "csb:test": "yarn csb:command run test && echo ALL GOOD || (echo NOT PASSED; exit 1)",
-        "csb:test-remote": "yarn workspace @river/sdk run test",
+        "csb:test-remote": "yarn workspace @river-build/sdk run test",
         "csb:turbo": "bash -c 'yarn run turbo run build test:ci $(yarn csb:list:filter)'",
         "csb:turbo-no-sdk": "bash -c 'yarn run turbo run build test:ci $(yarn csb:list:filter-without-sdk) --no-cache --force'",
         "csb:wait": "yarn csb:wait:entitlements",
         "csb:wait:entitlements": "wait-on https://localhost:5157/info --timeout=900000 --i=5000 --verbose",
         "csb:wait:no-entitlements": "wait-on https://localhost:5158/info --timeout=900000 --i=5000 --verbose",
         "drt:wait": "wait-on tcp:8008 --timeout=900000 --i=5000 --verbose",
-        "harmonyweb:build": "turbo build --filter harmonyweb",
         "install:foundry": "./scripts/foundry-up.sh && yarn forge install",
         "lint": "turbo lint",
         "postinstall": "husky install",
         "prepare": "husky install",
         "prettier:check": "prettier --check .",
         "prettier:fix": "prettier --write .",
-        "publish:local": "turbo publish:local",
-        "sampleapp:build": "turbo build --filter sample-app",
         "syncpack:check": "syncpack list-mismatches",
         "syncpack:fix": "syncpack fix-mismatches && syncpack format",
         "test": "turbo test",
-        "test:react": "vitest --project @river-build/react-sdk",
-        "test:build": "turbo test:build",
         "test:unit": "turbo test:unit -- --silent",
-        "worker:wait:stackup": "wait-on tcp:127.0.0.1:${PORT:-8686} --timeout=900000 --i=5000 --verbose",
-        "workers:build": "turbo build --filter gateway-worker --filter siwe-worker --filter token-worker --filter unfurl-worker --filter jwt-worker"
+        "test:react": "vitest --project @river-build/react-sdk",
```

Comment:
this should be "test:react-sdk"

Reply:
and second thought, i don't think we normally put the individual package tests at this level - can you just make sure that these tests run in the test:unit ci job? (I think we can add a test:unit to the react-sdk package that calls yarn test there.

Reply:
I'll address this in another PR -- I think it makes sense to drop vitest for now, need to change the react-sdk package to use jest.